quinn/src/tests.rs

```diff
@@ -25,7 +25,7 @@ fn echo_v4() {
 }
 
 #[test]
-#[cfg(any(target_os = "linux", target_os = "macos"))] // Dual-stack sockets aren't the default anywhere else.
```

Comment:
We can ditch this again before we merge it, it's just there to help me test the PR.

Reply:
Your call, pending Apple's response to that bug report.